apiserver/facades/client/action/action.go

```diff
@@ -308,10 +302,6 @@ func (a *ActionAPI) Cancel(arg params.Entities) (params.ActionResults, error) {
 		return params.ActionResults{}, errors.Trace(err)
 	}
 
-	if err := a.check.ChangeAllowed(); err != nil {
```

Comment:
I am not sure that we want to allow cancelling the action when the change block is on. I think we might want to have this check.

Reply:
@wallyworld thoughts?

Reply:
Yes we do. Starting and cancelling actions are orchestration and devops tasks. Not model changes.

Reply:
Per discussion in IRC, retaining change.